### PR TITLE
start command implementation

### PR DESCRIPTION
**Pull request by elvyyy**

### CODE REVIEW

**Review by pinkyfox — APPROVED** [review]

File: src/main/java/ru/tinderbook/bot/realt/resolver/update/MessageUpdateResolver.java [hunk 1]
```diff
@@ -5,20 +5,28 @@
...
+    private final UserMapper userMapper;
+
     public boolean canBeApplied(Update update) {
         return update.hasMessage();
     }
 
     @Override
     public SendMessage resolve(Update update) {
+        final var user = userMapper.toDbUser(update.getMessage().getFrom());
+        userService.saveUserOrGetExistingOne(user);
```
Comment (pinkyfox) [inline src/main/java/ru/tinderbook/bot/realt/resolver/update/MessageUpdateResolver.java:29]: I think, that mapping should be inside UserService.

Comment (elvyyy) [reply src/main/java/ru/tinderbook/bot/realt/resolver/update/MessageUpdateResolver.java]: Agree. Resolved

File: src/main/java/ru/tinderbook/bot/realt/service/UserService.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
      * @param user user to create
      * @return user
      */
-    public User saveUser(User user) {
+    public User saveUserOrGetExistingOne(User user) {
```
Comment (pinkyfox) [inline src/main/java/ru/tinderbook/bot/realt/service/UserService.java]: Rename to `saveIfAbsent`.

File: src/main/java/ru/tinderbook/bot/realt/resolver/update/MessageUpdateResolver.java [hunk 3]
```diff
@@ -17,16 +16,13 @@
...
-
     public boolean canBeApplied(Update update) {
         return update.hasMessage();
     }
 
     @Override
     public SendMessage resolve(Update update) {
-        final var user = userMapper.toDbUser(update.getMessage().getFrom());
-        userService.saveUserOrGetExistingOne(user);
+        userService.saveUserOrGetExistingOne(update.getMessage().getFrom());
```
Comment (pinkyfox) [inline src/main/java/ru/tinderbook/bot/realt/resolver/update/MessageUpdateResolver.java:25]: Store `update.getMessage()` in a local variable.

File: src/main/java/ru/tinderbook/bot/realt/service/UserService.java [hunk 4]
```diff
@@ -11,15 +12,18 @@
...
     /**
      * Creates user.
      *
      * @param user user to create
      * @return user
      */
-    public User saveUserOrGetExistingOne(User user) {
-        return userRepository.findById(user.getId())
-                .orElseGet(() -> userRepository.save(user));
+    public User saveUserOrGetExistingOne(org.telegram.telegrambots.meta.api.objects.User user) {
```
Comment (pinkyfox) [inline src/main/java/ru/tinderbook/bot/realt/service/UserService.java:23]: Rename to saveIfAbsent.

**Review by pinkyfox — APPROVED** [review]

**Pull request merged**